### ISSUE TITLE
New freerdp service won't run under localSystem

### ISSUE THREAD

**Issue by tmakaro** [cedrozor/myrtille#16]
Hi Cedric, have you been able to get freerdp(the new one) working as a service under local system? Wfreerdp.exe keeps crashing on me but when I changed the service to use the Administrator account it works fine.  

I hope that is not going to be a requirement. The old freerdp can use localSystem.

**Comment by cedrozor** [comment]
Hi Troy, it may be due to the (new) clipboard redirect feature which require read/write access to the local (gateway) clipboard, thus elevated rights. It is part of the remaining tests left to be done. I hope it's not the reason because i also would like freerdp to run under LocalSystem.

Can you enable the logs (debug) when you start a remote session? then check what it says before crashing.

Thanks :)

**Comment by tmakaro** [comment]
I'll give that a try after work. Thanks.

**Comment by tmakaro** [comment]
I turned logging on but no log was created. I thought maybe I did something wrong so I restarted the service under administrator and this time the log was created so debug isn't helping us.

I don't think the problem is the clipboard because I commented it out of the wfreerdp.exe arguments and it still wouldn't start under localSystem. Under Administrator the log shows:

> [3396:000010a0] [INFO][com.freerdp.client.windows] - GDI rendering: hardware
[3396:000010a0] [INFO][com.freerdp.client.windows] - Clipboard redirect: off
[3396:000010a0] [INFO][com.freerdp.client.myrtille] - wf_myrtille_connect: connected session 55
[3396:000014f0] [INFO][com.freerdp.client.myrtille] - ProcessInputsPipe: ReadFile succeeded: Hello client
[3396:000014f0] [INFO][com.freerdp.client.myrtille] - ProcessInputsPipe: ReadFile succeeded: MMO946-306,
[3396:000014f0] [INFO][com.freerdp.client.myrtille] - ProcessInputsPipe: ReadFile succeeded: MMO945-306,
[3396:000014f0] [INFO][com.freerdp.client.myrtille] - ProcessInputsPipe: ReadFile succeeded: MMO945-305,
